package.json

```diff
@@ -42,7 +41,7 @@
     "@nrwl/vite": "^15.6.3",
     "@nrwl/web": "15.6.3",
     "@nrwl/workspace": "15.6.3",
-    "@tacc/core-styles": "^2.6.2",
+    "@tacc/core-styles": "^2.14.0",
```

Comment:
Updating Core-Styles lets me use many fixes I made because of this PR.

_The side effect should only be that the three lightest global accent colors become a bit darker._